.github/ISSUE_TEMPLATE/blank_issue.yml

```diff
@@ -0,0 +1,11 @@
+name: 📄 Blank Issue
+description: If you know what you're doing (especially collaborators)
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please remember to add appropriate labels, assignees, projects, etc.
```

Comment:
I think I prefer this custom template, even if the sorting is not quite what we want.

Reply:
:+1: Should I call it "z_blank_issue.yml" for sorting? A bit gross, but I think the user experience is more important here

Reply:
Oh... hmm.. if the name of the _file_ controls sorting, but we can still set the `name` in yml to whatever we want independent of that, then sure... I guess :P

Reply:
That's my guess. Not sure though

Reply:
Indeed it does:

![image](https://user-images.githubusercontent.com/5732088/180095889-59932172-acbd-4e95-9bf6-2c3f90eb7ab6.png)